pom.xml

```diff
@@ -22,56 +22,84 @@
     ~ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.osiam</groupId>
-        <artifactId>osiam-server-parent</artifactId>
-        <version>2.1-SNAPSHOT</version>
-    </parent>
-
+    <groupId>org.osiam</groupId>
     <artifactId>osiam-resource-server</artifactId>
```

Comment:
I can think of an impact for people who are already using the new 'classes' JAR (as described here: https://github.com/osiam/server/issues/266). But as it is really new stuff and also not officially supported yet, maybe it's still ok to change this now.